smarts/core/coordinates.py

```diff
@@ -247,11 +258,13 @@ class Pose:
     """A pair of position and orientation values."""
 
     # TODO: these should be np.ndarray
-    position: Sequence  # [x, y, z]
-    orientation: Sequence  # [a, b, c, d] -> a + bi + cj + dk = 0
+    position: Union[NamedTuple, Tuple, np.ndarray]  # [x, y, z]
+    orientation: Union[
+        NamedTuple, Tuple, np.ndarray
+    ]  # [a, b, c, d] -> a + bi + cj + dk = 0
```

Comment:
I set these to be a tuple or numpy array.

Reply:
I thought here https://github.com/huawei-noah/SMARTS/pull/1276/files we only want numpy arrays?